src/app/(checkin)/checkin/validates/page.client.tsx

```
@@ -13,8 +13,18 @@ import { useTranslate } from '@/providers/I18n/client'
+  attemptAutoSelection,
+  type EventWithSchedules
+} from '@/lib/checkin/autoEventSelection'
+import {
+  getCachedEventSelection,
+  setCachedEventSelection,
+  clearExpiredCache
+} from '@/lib/checkin/eventSelectionCache'
+import { format } from 'date-fns'
 
-interface CheckinHistoryProps {}
+interface CheckinHistoryProps { }
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)